testplan/testing/multitest/base.py

```diff
@@ -340,6 +340,7 @@ def dry_run(self, status=None):
                 testcase_report = TestCaseReport(
                     name=testcase.__name__,
```

Comment:
Here and below you should set `name=testcase.__display_name__` and not set any `display_name` parameter in the report.